docs/dev/build_env.rst

```diff
@@ -130,15 +130,15 @@ for some architectures, you can install those via
 Qt
 ''
 
-The simplest way is to install whichever version of Qt is appropriate for your distro (18.04::Qt5.9; 20.04::Qt5.12):
+The simplest way is to install whichever version of Qt is appropriate for your distro (20.04::Qt5.12):
     - :command:`apt install qt5-default` (not necessary for liblsl)
     
 However, if your app requires a newer version of Qt then the easiest way to install it is with `aqtinstall <https://aqtinstall.readthedocs.io/en/latest/>`__:
     - :command:`sudo -i`
     - :command:`apt install python3-pip`
     - :command:`pip3 install aqtinstall`
 
-The newest version that will work with Ubuntu 18.04 is Qt 5.15.2:
+The newest version that will work with Ubuntu 20.04 is Qt 5.15.2:
```

Comment:
Qt6 works with Ubuntu 20.04 if you're willing to install another ppa
```
sudo add-apt-repository -y ppa:okirby/qt6-backports
sudo apt-get install -y cmake qt6-base-dev
```

Reply:
IMHO Qt is something so fundamental it should be provided (and supported) by the OS vendor. On the other hand, getting Qt5 on Windows is more complicated than it should be and there's no reason for the Windows-only apps to stay on Qt5...

Reply:
So is that a yes or no on Qt6 for Ubuntu 20.04? If yes for Qt6, is this install method acceptable? 

If we update our docs and move our Actions runners to Qt6 on Ubuntu 20.04, then we don't have any OS-driven Qt5 dependencies anymore.

There are things like QGamepad that are [not available in Qt6](https://lists.qt-project.org/pipermail/development/2021-March/041125.html), but then I think it's OK to expect that individual application to provide Qt5 install instructions.

Reply:
It's a "as an Ubuntu 20.04 user I'd hate it, but it's unrealistic to require everyone else to keep it around for testing".